### PR TITLE
chore: migrate scheduler logger to tracing

### PR DESCRIPTION
**Pull request by AlexandreBrg**
Closes # 

## 📑 Description

* Migrate scheduler logger to tracing
* Riklet: use conventional methods to kick start the logger

## ✅ Checks

- [ ] My pull request adheres to the code style of this project
- [ ] My code is documented
- [ ] I provided unitary tests or procedure to test my code
- [ ] My PR have a clear description of what my code is supposed to do

### CODE REVIEW

File: riklet/src/core.rs [hunk 1]
```diff
@@ -138,8 +138,8 @@ impl Riklet {
     }
 
     pub async fn run(&mut self) -> Result<()> {
-        event!(Level::INFO, "Riklet is running.");
         self.start_metrics_updater();
+        event!(Level::INFO, "Riklet is running.");
```
Comment (MaloPolese) [inline riklet/src/core.rs:142]:
```suggestion
        info!("Riklet is running.");
```

File: scheduler/src/main.rs [hunk 2]
```diff
@@ -6,7 +6,6 @@ use crate::config_parser::ConfigParser;
 use crate::grpc::GRPCService;
 use crate::state_manager::{StateManager, StateManagerEvent};
 use env_logger::Env;
```
Comment (MaloPolese) [inline scheduler/src/main.rs:8]: remove unused import

File: scheduler/src/main.rs [hunk 3]
```diff
@@ -16,6 +15,7 @@ use scheduler::{Controller, SchedulerError, Worker, WorkerRegisterChannelType};
 use std::default::Default;
```
Comment (MaloPolese) [inline scheduler/src/main.rs:15]: remove unused import

File: scheduler/src/main.rs [hunk 4]
```diff
@@ -296,7 +296,9 @@ impl Manager {
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let config = ConfigParser::new()?;
-    env_logger::Builder::from_env(Env::default().default_filter_or(&config.verbosity_level)).init();
+    let subscriber = tracing_subscriber::fmt().compact().finish();
```
Comment (MaloPolese) [inline scheduler/src/main.rs:299]: if the logger is the same for `rikctl` / `controller` / `scheduler` / `riklet` why not move the logger initialisation inside a global common lib ?

**Review by MaloPolese — CHANGES_REQUESTED** [review]

File: scheduler/src/main.rs (hunk 4 above)
Comment (AlexandreBrg) [reply scheduler/src/main.rs:299]: For now, it's to keep the governance of logger into the project, due to how critical it is.

Comment (MaloPolese) [reply scheduler/src/main.rs:299]: il will to it in https://github.com/rik-org/rik/issues/69

**Review by MaloPolese — APPROVED** [review]

**Pull request merged**